services/rest/websocket/connection/table.go

```diff
@@ -44,11 +44,26 @@ func (t *Table) Store(c identification.Identifier) error {
 	if _, ok := t.connMap[c]; ok {
 		return errConnDuplicated
 	}
-	t.connMap[c] = struct{}{}
+	t.connMap[c] = make(map[string]struct{})
 	t.counter[c.Group] = t.counter[c.Group] + 1
 	return nil
 }
 
+func (t *Table) StoreEvent(c identification.Identifier, reqId string) {
+	t.m.Lock()
+	defer t.m.Unlock()
+	if _, ok := t.connMap[c]; ok {
+		t.connMap[c][reqId] = struct{}{}
+	}
+}
+
+func (t *Table) HasEvent(c identification.Identifier, reqId string) bool {
```

Comment:
This function name could also reflect hasBatchId()

Reply:
Updated